test/unit/model/jdk-install-test.js

```diff
@@ -162,7 +162,26 @@ describe('JDK installer', function() {
       });
     });
 
+    it('should create deafult empty callback if not provided', function() {
+      let jdk = new JdkInstall(installerDataSvc, 'url', 'file');
+      mockDetectedJvm('1.8.0_1');
+      try {
+        jdk.detectExistingInstall();
+      } catch (exception)  {
+        expect.fail('Did not created default empty callback');
+      }
+    });
 
+    it('should check for available msi installtion', function(done) {
+      mockDetectedJvm('1.8.0_1');
+      let jdk = new JdkInstall(installerDataSvc, 'url', 'file');
+      jdk.findMsiInstalledJava.restore();
+      return jdk.detectExistingInstall(function() {
+        expect(executeFileStub).to.have.been.calledOnce;
+        expect(writeFileStub).to.have.been.calledOnce;
```

Comment:
I've looked into the test failures on CI and found this particular test timed out because the assertions are not met and the eventual error is consumed by the callback.

The root of the problem is on this line: https://github.com/redhat-developer-tooling/developer-platform-install/blob/master/browser/model/jdk-install.js#L102
The tests are being run on a RHEL machine and these methods are only run on Windows. In the spirit of the unit tests running on more platforms, I'd request this piece of code be executed for Windows only.